aten/src/ATen/native/native_functions.yaml

```diff
@@ -711,7 +711,7 @@
 
 - func: gesv(Tensor self, Tensor A) -> (Tensor, Tensor)
 
-- func: gesv_out(Tensor solution, Tensor lu, Tensor self, Tensor A) -> (Tensor, Tensor)
+- func: gesv_out(Tensor result0, Tensor result1, Tensor self, Tensor A) -> (Tensor, Tensor)
```

Comment:
same here, please use the more informative names

Reply:
That's weird, I renamed these earlier and the codegen wouldn't allow me to pass `Tensor&` instead of `const Tensor&` if I didn't call them `result0, result1`, but I changed them back now and it seems to work...